core/flyout_base.js

```diff
@@ -426,77 +426,60 @@ Blockly.Flyout.prototype.hide = function() {
 
 /**
  * Show and populate the flyout.
- * @param {!Array|!NodeList|string} xmlList List of blocks to show.
+ * @param {!Array|!NodeList|string} contentInfo List of contents to display in
```

Comment:
Can this cover the different possible options for contentInfo. I know it's mentioned further below in the comments, but it's good to have that stated in the parameter.
Also, an Array of what? Array of NodeList only, Array of JSON, both?